### ISSUE TITLE
Trojan detected in the jar file

### ISSUE THREAD

**Issue by zasnool** [GTNewHorizons/Hodgepodge#344]
![Untsditled](https://github.com/GTNewHorizons/Hodgepodge/assets/100788360/42534b04-6f06-4759-9182-e95411f54d00)
both github version and the modrinth version 2.4.34

**Comment by Caedis** [comment]
https://www.virustotal.com/gui/file/54e9508fc1cab8cad04ad413eb6ea0b3fd923b3ee596f5e7cb0fa52153cec609?nocache=1

Feel free to upload the jar to virustotal yourself

**Comment by zasnool** [comment]
Yes, virustotal doesn't find anything. Windows defender still finds a trojan thoguh. Must be a false positive.

**Comment by TheInfamousAlk** [comment]
Happens to me too. While not a problem because I know it's safe, it may cause problems if windows starts deleting/quarantining this file en masse if more people install it, especially if it's included in a stable release of GTNH. Modpacks could just start breaking left and right and of course windows won't do anything about it.
